ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/TransactionMetaData.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+
+@DoNotImplement
+@CordaSerializable
+interface TransactionMetaData{
+    operator fun get(key: String): Any?
+    val entries: Set<Map.Entry<String, Any>>
+}
+
+const val TRANSACTION_META_DATA_LEDGER_MODEL_KEY = "ledgerModel"
+const val TRANSACTION_META_DATA_LEDGER_VERSION_KEY = "ledgerVersion"
+const val TRANSACTION_META_DATA_CPK_IDENTIFIERS_KEY = "cpkIdentifiers"
```

Comment:
You could omit the `TRANSACTION_META_DATA_` part and put them in a companion object inside `TransactionMetaData` maybe?

```kotlin
interface TransactionMetaData{
    operator fun get(key: String): Any?
    val entries: Set<Map.Entry<String, Any>>

    companion object {
      const val LEDGER_MODEL_KEY = "ledgerModel"
      const val LEDGER_VERSION_KEY = "ledgerVersion"
      const val CPK_IDENTIFIERS_KEY = "cpkIdentifiers"
    }
}
```

Then the constants would be namespaced under `TransactionMetaData`.

Reply:
Thanks. Good idea. I have refactored it.